AUTHORS.rst

```diff
@@ -110,6 +110,8 @@ Integrated Software for Imagers and Spectrometers Contributors
 - Milazzo, Moses
 - Miller, David
 - Miller-Ribelin, Elizabeth
+- Nelson, Gavin (United States Geological Survey, Astro Geology
```

Comment:
This is consistent with the existing entries, but we should consider changing "Astro Geology" to Astrogeology